clang/include/clang/Basic/BuiltinsX86_64.def

```diff
@@ -21,87 +21,87 @@
 #  define TARGET_HEADER_BUILTIN(ID, TYPE, ATTRS, HEADER, LANG, FEATURE) BUILTIN(ID, TYPE, ATTRS)
 #endif
 
-TARGET_HEADER_BUILTIN(_BitScanForward64, "UcUNi*ULLi", "nh", "intrin.h", ALL_MS_LANGUAGES, "")
-TARGET_HEADER_BUILTIN(_BitScanReverse64, "UcUNi*ULLi", "nh", "intrin.h", ALL_MS_LANGUAGES, "")
+TARGET_HEADER_BUILTIN(_BitScanForward64, "UcUNi*UOi", "nh", "intrin.h", ALL_MS_LANGUAGES, "")
```

Comment:
These are only available in MSVC compatbility mode. Do they need to be touched for OpenCL?

Reply:
@topperc I didn’t know that. Thank you for pointing out.